build.sbt

```diff
@@ -34,6 +34,7 @@ lazy val core = project
   .settings(Dependencies.settings:_*)
   .settings(Format.settings:_*)
   .settings(Publish.settings:_*)
+  .settings(Publish.extraReleaseStep:_*)
```

Comment:
This extraReleaseStep is to update the readme with the new version. So it doesn't matter which sub-project it goes to.